### PR TITLE
added "Upload from local"

### PR DESCRIPTION
**Pull request by andrijapanicsb**
added "Upload from local" explanation

### CODE REVIEW

File: source/adminguide/templates.rst [hunk 1]
```diff
@@ -272,6 +273,13 @@ To upload a template:
       the Featured Templates list. Only an administrator can make a
       template Featured.
 
+Uploading Templates from local computer
+-------------------------------------------
+
+It's also possible to upload already prepared template from your local computer.
+Steps are similar as when Uploading template from remote HTTP server, except that you need to choose local template file from your PC.
+For this feature to work, your SSVMs must be supporting HTTPS (for more info please visit `“Using a SSL Certificate for the Console Proxy”
```
Comment (PaulAngus) [inline source/adminguide/templates.rst]: Are you sure that this is true?
SSVM is a client not a server in this instance.
There is a bug regarding remote https not being recogniased
https://github.com/apache/cloudstack/issues/3172

File: source/adminguide/templates.rst [hunk 2]
```diff
@@ -272,6 +273,13 @@ To upload a template:
       the Featured Templates list. Only an administrator can make a
       template Featured.
 
+Uploading Templates from local computer
```
Comment (PaulAngus) [inline source/adminguide/templates.rst]: "Article" missing in sentence
```suggestion
Uploading Templates from a local computer
```

File: source/adminguide/templates.rst [hunk 3]
```diff
@@ -272,6 +273,13 @@ To upload a template:
...
+-------------------------------------------
+
+It's also possible to upload already prepared template from your local computer.
```
Comment (PaulAngus) [inline source/adminguide/templates.rst]: "Article" missing in sentence
```suggestion
It's also possible to upload an already prepared template from your local computer.
```

File: source/adminguide/templates.rst [hunk 4]
```diff
@@ -272,6 +273,13 @@ To upload a template:
...
+Steps are similar as when Uploading template from remote HTTP server, except that you need to choose local template file from your PC.
```
Comment (PaulAngus) [inline source/adminguide/templates.rst]: "Articles" missing in sentence
```suggestion
The steps are similar as when Uploading a template from a remote HTTP server, except that you need to choose a local template file from your PC.
```

File: source/adminguide/templates.rst [hunk 5]
```diff
@@ -201,8 +201,9 @@ you can create a template directly from any snapshot through the
 CloudStack UI.
 
 
-Uploading Templates
--------------------
+Uploading Templates from remote HTTP server
```
Comment (PaulAngus) [inline source/adminguide/templates.rst]: "Article" missing in sentence
```suggestion
Uploading Templates from a remote HTTP server
```

File: source/adminguide/templates.rst (hunk 1 above)
Comment (andrijapanicsb) [reply source/adminguide/templates.rst]: https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=39620237 says "The Apache web server will only accept upload requests over HTTPS" and yes, tested this (for other reasons) and found it's NOT working when your SSVM doesn't have SSL configured (captured logs confirming HTTPS url "was not found" kind of thing - that is reason I opened this PR.
Confirmed it works if SSVM is SSL enabled.

**Pull request merged**